.github/workflows/ci.yml

```diff
@@ -35,9 +35,10 @@ jobs:
       uses: actions/checkout@v3
     - if: env.RUNNER_ENVIRONMENT != 'github-hosted'
       uses: dtolnay/rust-toolchain@stable
+    - name: Installing the toolchain
+      run: make toolchain
```

Comment:
I _think_ this is a no-op on custom runners (because the `rust-toolchain` action above should install the correct toolchain) but it won't hurt, it just might not do anything.